app/src/main/java/com/lastaoutdoor/lasta/ui/screen/favorites/FavoritesScreen.kt

```diff
@@ -26,13 +27,20 @@ import com.lastaoutdoor.lasta.ui.screen.discover.ActivitiesDisplay
 fun FavoritesScreen(
     isLoading: Boolean,
     activities: List<Activity>,
+    updateFavorites: () -> Unit,

```

Comment:
As a convention I think it would be nice to put all lambda parameters at the end of the parameter list of a function

Reply:
You're right, will do that!